pkg/bugtool/bugtool.go

```diff
@@ -43,6 +43,8 @@ type InitInfo struct {
 	MetricsAddr string `json:"metrics_address"`
 	GopsAddr    string `json:"gops_address"`
 	MapDir      string `json:"map_dir"`
+	Path        string `json:"path"` // Environment variable $PATH
```

Comment:
I don't think this is needed (or used). Can we remove it?

Reply:
done